src/Config.ts

```diff
@@ -22,6 +22,44 @@ export class SocketOptions {
     //TO-DO
 }
 
+export interface SslOptions {
```

Comment:
I would prefer this as a class rather than interface. For JS use this does not matter but if somebody uses this through Typescript they would like to instentiate an object of this and assign relevant parameters. This is meant to be used as an object, afterall, right?